src/js/api.ts

```diff
@@ -15,7 +15,7 @@ import { setStdin, setStdout, setStderr } from "./streams";
 API.loadBinaryFile = loadBinaryFile;
 
 /**
- * An alias to the Python :py:mod:`pyodide` package.
+ * An alias to the Python `pyodide </usage/api/python-api.html>`_ package.
```

Comment:
Are you sure this works? It would be better to link to  a reference here. I think it wouldn't work when deployed under the /en/stable prefix in RTD.

Otherwise LGTM, thanks!

I'll fix this and merge, so we can use it in other PRs.